run-build-functions.sh

```diff
@@ -55,11 +55,19 @@ mkdir -p $NETLIFY_CACHE_DIR/.composer
 mkdir -p $NETLIFY_CACHE_DIR/.gimme_cache/gopath
 mkdir -p $NETLIFY_CACHE_DIR/.gimme_cache/gocache
 mkdir -p $NETLIFY_CACHE_DIR/.wasmer/cache
+mkdir -p $NETLIFY_CACHE_DIR/.cargo/registry
+mkdir -p $NETLIFY_CACHE_DIR/repo/target
```

Comment:
This should be replaced by:

```
 mkdir -p $NETLIFY_REPO_DIR/target 
```

Reply:
i think it can be removed entirely. `cargo` will create `target` if it doesn't exist.

Reply:
Fixed in 5cd8e76